mongodb-storage/src/main/java/io/lumeer/mongodb/MongoDbStorageDialect.java

```diff
@@ -67,6 +69,10 @@ public DataDocument addRecentlyUsedDocumentQuery(final String metadataCollection
                               .append("$slice", listSize))));
    }
 
+   public String sortStringForSearch(final String field, final int order) {
+      return MongoUtils.convertBsonToJson(Sorts.descending(field));
```

Comment:
What is this method good for? I cannot see it used anywhere and it seems to ignore `order` parameter.